composer.json

```diff
@@ -17,11 +17,25 @@
         "psr-4": { "Omnipay\\WestpacPaywayRest\\" : "src/" }
     },
     "require": {
-        "omnipay/common": "^3"
+        "php": "^8.1",
```

Comment:
Should this be 8.2?

Reply:
The `platform` constraint below this will ensure that composer only installs dependencies compatible with PHP 8.2.  This is just saying we want to consider packages that are marked as compatible >= 8.1 and  < 9.0